plugins/circleci/src/circleci-config.ts

```diff
@@ -432,47 +432,48 @@ const generateWorkflowJobs = (
       [prefixedName]: merge(
         executorParameter,
         {
-          requires: job.requires?.map((required) => {
-            const getRequiredData = (): [string, ExecutorCount] => {
-              if (required === 'checkout') {
-                return [required, 'none']
+          requires:
```

Comment:
hmm, don't know why Prettier has made these changes? this part of the diff can be ignored, the actual changes are [below](https://github.com/Financial-Times/dotcom-tool-kit/pull/1127/files#diff-f157d7392cc3db08e0c5b213317a296767e475221a5be30ae3bcc8b94fa7613fR594-R595)